R/ecdc.R

```diff
@@ -20,37 +20,29 @@
 #' get_ecdc_cases(countries = "France")
```

Comment:
example no longer works so can be removed

Reply:
fixed